components/movementsensor/adxl345/adxl345.go

```diff
@@ -1,7 +1,18 @@
 // Package adxl345 implements the MovementSensor interface for the ADXL345 accelerometer attached
-// to the I2C bus of the robot (the chip supports communicating over SPI as well, but this package
-// does not support that interface). The manual for this chip is available at:
+// to an I2C bus on the robot (the chip supports communicating over SPI as well, but this package
+// does not support that interface). The datasheet for this chip is available at:
 // https://www.analog.com/media/en/technical-documentation/data-sheets/adxl345.pdf
+
+// We support reading the accelerometer data off of the chip. We do not yet support using the
+// digital interrupt pins to notify when interesting things happen (freefall, collisions, etc.). 
+
+// Because we only support I2C interaction, the CS pin must be wired to a high signal (which tells
+// the chip which communication interface to use). The chip has two possible I2C addresses, which
+// can be selected by wiring the SDO pin to either hot or ground:
+//   - if SDO is wired to ground, it uses the default I2C address of 0x53
+//   - if SDO is wired to hot, it uses the alternate I2C address of 0x1D
```

Comment:
I refer to this as either the "high signal" (line 9) or "hot" (line 13), and feel like it should be one or the other but not both (or maybe both is okay? I dunno). What do you think?